src/AbstractOperations/multiary_operations.jl

```diff
@@ -31,8 +31,8 @@ function define_multiary_operator(op)
         # "Function, or Field"
         local FuFi = Union{Function, AbstractField}
```

Comment:
Might opt for the more descriptive name `FunctionOrField` and if you need a short name for concise multiple dispatch, then we can define
```julia
const FOF = FunctionOrField
```
but maybe this is bad as it pollutes the namespace. I'd still vote for `FunctionOrField` over `FuFi`.

Reply:
`FuFi` is intended to be concise. We can avoid an alias altogether if we aren't concerned about being concise. I'll make that change.